src/modules/job-manager/plugins/limit-duration.c

```diff
@@ -94,7 +94,7 @@ static double queues_lookup (zhashx_t *queues, const char *name)
 static int queues_insert (zhashx_t *queues, const char *name, double duration)
 {
     if (zhashx_insert (queues, name, &duration) < 0) { // dups duration
-        errno = ENOMEM;
+        errno = EEXIST;
```

Comment:
fix the caller too - it hardwires "out of memory".

Reply:
ahh good catch on this one and one below.  Having the `queues_insert()` just return void is probably for best.